docs/_posts/2017-09-08-520-release.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: blogpost
+title:  "5.2.0 Release"
+section_id: blog
+date:   2017-09-08 13:30:00
+author: Alexander Hernandez
+thumbnail: ahernandez.jpg
+---
+
+Hello Everyone!
+
+I’m excited to bring you the news of OpenGEE’s 5.2.0 Release! We’ve been working hard to create the first official release since the project was opened sourced. Here are some highlights of the changes:
+
+**Open Sourcing**: Open GEE 5.2.0 includes a number of changes that were necessary for the open sourcing effort. This includes several library replacements, library unbundling, removal of obsolete code and libraries, changes to support new compilers, updates to support Git and GitHub, and several documentation changes.
+
+**New Installers**: Previous GEE installers used proprietary tools that are not appropriate for open source projects. Open GEE 5.2.0 includes new installers that run as bash scripts. There are also new build and install scripts for the Portable Server. We will continue to improve the installers, create distribution libraries and add additional platform support like CentOS/RHEL 6.x in future versions.
+
+**New Tutorial Files**: Images used in the Fusion tutorial are now provided in both GeoTIFF and JPEG2000 format. In addition, the tutorial files are stored and downloaded separately from the rest of the Open GEE code.
+
+**JPEG2000**: JPEG2000 was originally supported in the commercial version with proprietary libraries. In Open GEE 5.2.0 JPEG2000 support is back using open source libraries that support the image format.
```

Comment:
The last part of this sentence -- "that support the image format" -- seems unnecessary to me.

Reply:
Also OpenGEE instead of "open GEE" in last line.